lib/cube_browser/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 class Pyplot(object):
     """"""
-    def __init__(self, cube, coords, **kwargs):
+    def __init__(self, cube, coords, axes, **kwargs):
```

Comment:
@marqh `axes` are definitely an argument, but `coords` (I think) should be a `kwarg` akin to `iris` (a change to come!)

So, could we swap the order here i.e.
```python
def __init__(self, cube, axes, coords, **kwargs)
```